README.md

```diff
@@ -544,6 +544,25 @@ The version number of the agent build. The Server can use this information for
 example to decide if it wants to offer a package of a different version to the
 agent via AgentPackageAvailable message.
 
+#### agent_attributes
+
+Attributes that describe the agent and the environment it runs in. Keys/values
+are according to OpenTelemetry semantic conventions, see:
+https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/resource/semantic_conventions
+
+The following attributes SHOULD be included:
+- os.type, os.version - to describe where the agent runs.
+
+The following attributes are recommended to be included:
```

Comment:
What's the difference between `SHOULD` and `recommended`?  Since `SHOULD` is not `MUST` isn't it a recommendation instead of a requirement?

Reply:
I guess what I tried to express is that host.*, cloud.*, etc may not be known to the Agent. I probably should rephrase and say "The following attributes SHOULD to be included if they are available" or perhaps even that is unnecessary since "if available" is implied anyway.

Reply:
Removed the distinction, now it is just one list with a SHOULD keyword.